Dispatcher/UI/Screens/ArticleDetailsPage/ArticleDetailsView.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  ArticleDetailsView.swift
+//  Dispatcher
+//
+//  Created by Hadar Basson on 27/08/2023.
+//
+
+import UIKit
+import Kingfisher
+
+class ArticleDetailsView: UIView {
+    
+    let article: NewsArticle?
+    let customColor = UIColor(named: "your-article-details-color")
+    
+    
+    lazy var image: UIImageView = {
+        let image = UIImageView(image: UIImage(named: "metaData-img"))
+        image.translatesAutoresizingMaskIntoConstraints = false
+        if(article?.urlToImage != nil) {
+            let url = URL(string: article?.urlToImage ?? "")
+            image.kf.setImage(with: url)
+            return image
+        }
+        image.image = UIImage(named: "no-image")
+        return image
+    }()
+    
+    lazy var date: UILabel = {
+        let dateLable = UILabel()
+        dateLable.translatesAutoresizingMaskIntoConstraints = false
+        dateLable.text=convertDateString(article?.publishedAt ?? "")
+        let customColor = UIColor(named: "date-color")
+        dateLable.textColor = customColor
+        dateLable.font = UIFont(name: "Roboto-Regular", size: 14)
+        return dateLable
+    }()
+    
+    
+    lazy var title: UILabel = {
+        let titleLable = UILabel()
+        titleLable.translatesAutoresizingMaskIntoConstraints = false
+        titleLable.text = article?.title
+        titleLable.font = UIFont.boldSystemFont(ofSize: 18)
+        titleLable.numberOfLines = 4
+        return titleLable
+    }()
+    
+
+    lazy var author: UILabel = {
+        let authorLabel = UILabel()
+        authorLabel.translatesAutoresizingMaskIntoConstraints = false
+        authorLabel.text = article?.author
+        let customColor = UIColor(named: "date-color")
+        authorLabel.textColor = customColor
+        authorLabel.font = UIFont(name: "Roboto-Regular", size: 14)
+        authorLabel.numberOfLines = 2
+       
+        return authorLabel
+    }()
+    
+    lazy var content: UITextView = {
+        let textView = UITextView()
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.text = article?.content
+        let customColor = UIColor(named: "content-color")
+        textView.textColor = customColor
+        textView.font = UIFont.systemFont(ofSize: 16)
+        textView.isScrollEnabled = true
+        textView.textAlignment = .left
+        return textView
+    }()
+    
+    
+    init(frame: CGRect, article: NewsArticle?) {
```

Comment:
we have no need for this init

use a regular init()
and just call super.init(frame: .zero) inside of it